cmd/logviewer/main.go

```diff
@@ -1,29 +1,39 @@
 package main
 
 import (
+	"encoding/hex"
 	"fmt"
 	"net/url"
 	"os"
 	"os/signal"
 	"path/filepath"
-	"regexp"
 	"runtime"
-	"strings"
 	"syscall"
 	"time"
 
-	"github.com/ElrondNetwork/elrond-go/cmd/logviewer/testing"
 	"github.com/ElrondNetwork/elrond-go/core"
 	"github.com/ElrondNetwork/elrond-go/logger"
+	protobuf "github.com/ElrondNetwork/elrond-go/logger/proto"
```

Comment:
The LogLineMessage structure can also be used with json marshalizer. Because it was autogenerated in proto folder I used it as it is (without creating a secondary wrapper). I used protobuf for performance reasons here. Could have used json Marshalizer as well.

Reply:
Or capnp for performance since we already use it in the project

Reply:
Added capnp marshaling capabilities. Added tests for that :D. The app still use protobuf.